dw-general-ami/generic_packer_template.json.j2

```diff
@@ -9,7 +9,7 @@
       "source_ami_filter": {
         "filters": {
           "virtualization-type": "{{ event['source_ami_virt_type'] or 'hvm' }}",
```

Comment:
should this one be fixed to hvm too?

Reply:
Guess that question could apply to the ones below too?